src/index.html

```diff
@@ -208,6 +209,7 @@
             <button class="btn clearLog-btn" onclick="javascript:document.getElementById('log-textarea').value = ''">clear log</button>
             <span>Dump to log:</span>
             <button class="btn dumpState-btn">state</button>
+            <button class="btn dumpIterationLog-btn">iteration log</button>
```

Comment:
I think I've removed iteration log dump button. maybe it sneaked back in a merge you did?